crates/staged-sync/src/config.rs

```diff
@@ -153,6 +153,12 @@ mod tests {
     use super::Config;
     #[test]
     fn can_serde_config() {
+        // Delete config file if it exists, as it can be outdated and
+        // make the test fail. It will be then created.
+        let config_path = confy::get_configuration_file_path("test", None).unwrap();
```

Comment:
I see `get_configuration_file_path`

> Load an application configuration from disk

we should not use that here and instead use a tmpdir as @rkrasiuk suggested

we also want an additional test for breaking changes with the config itself, because this will become an issue with `confy::load` in the CLI